jobs/picard.py

```diff
@@ -37,42 +42,47 @@ def get_intervals(
     that, but Hail Batch is not dynamic and have to expect certain number of output
```

Comment:
In line 41 (GitHub doesn't let make a suggestion there), "unpredicted" should probably be "unpredictable".